### PR TITLE
MMT-1550

### PR DESCRIPTION
**Pull request by lhlyles**
I updated the text for this ticket. When you all have time I would like to do a pull request for these changes.

Thanks
Monica

### CODE REVIEW

File: spec/features/users/user_login_spec.rb [hunk 1]
```diff
@@ -189,10 +189,10 @@
 
     it 'displays both login options' do
       expect(page).to have_link('Login with Launchpad', href: login_path(login_method: 'launchpad'))
-      expect(page).to have_link('Login with Earthdata Login', href: login_path(login_method: 'urs'))
+      expect(page).to have_link('Login with Earthdata Login', href: login_path(login_method: 'urs))
```
[the remark this answers is not in the document]
Comment (lhlyles) [reply spec/features/users/user_login_spec.rb]: Updated in branch. I will try to see how to update this pull request. Thanks for catching this Hoan

File: spec/features/users/user_login_spec.rb [hunk 2]
```diff
@@ -112,27 +112,27 @@
           real_login(method: 'launchpad', associated: false)
         end
 
-        it 'prompts the user to log into URS to link their accounts' do
-          expect(page).to have_content('Redirecting to URS')
-          expect(page).to have_content('It appears you do not have a URS account linked with your Launchpad account.')
-          expect(page).to have_content('Please click the button below to log in to the URS account you listed on your NAMS request and would like to associate with your Launchpad account.')
+        it 'prompts the user to log into Earthdata Login to link their accounts' do
+          expect(page).to have_content('Redirecting to Earthdata Login')
+          expect(page).to have_content('It appears you do not have a Earthdata Login account linked with your Launchpad account.')
```
Comment (charleshuang80) [inline spec/features/users/user_login_spec.rb]: I think this test is failing in Bamboo. This was the error message: 
```
RSpec::Expectations::ExpectationNotMetError: expected to find text "It appears you do not have a Earthdata Login account linked with your Launchpad account." in "Skip to main content Metadata Management Tool Help JavaScript is disabled! JavaScript must be enabled for some features to work. Redirecting to Earthdata Login It appears you do not have an Earthdata Login account linked with your Launchpad account. MMT and CMR permissions (ACLs) are based on your Earthdata Login account. Please click the button below to log in to the Earthdata Login account you listed on your NAMS request and would like to associate with your Launchpad account. Earthdata Login v (unknown) NASA Official: Stephen Berrick FOIA NASA Privacy Policy USA.gov"
Failure/Error: expect(page).to have_content('It appears you do not have a Earthdata Login account linked with your Launchpad account.')
  expected to find text "It appears you do not have a Earthdata Login account linked with your Launchpad account." in "Skip to main content Metadata Management Tool Help JavaScript is disabled! JavaScript must be enabled for some features to work. Redirecting to Earthdata Login It appears you do not have an Earthdata Login account linked with your Launchpad account. MMT and CMR permissions (ACLs) are based on your Earthdata Login account. Please click the button below to log in to the Earthdata Login account you listed on your NAMS request and would like to associate with your Launchpad account. Earthdata Login v (unknown) NASA Official: Stephen Berrick FOIA NASA Privacy Policy USA.gov"
```

File: app/views/users/prompt_urs_association.html.erb [hunk 3]
```diff
@@ -3,8 +3,8 @@
     <div class="content">
       <section>
         <fieldset>
-          <h2>Redirecting to URS</h2>
-          <p>It appears you do not have a URS account linked with your Launchpad account. MMT and CMR permissions (ACLs) are based on your URS account. Please click the button below to log in to the URS account you listed on your NAMS request and would like to associate with your Launchpad account.</p>
+          <h2>Redirecting to Earthdata Login</h2>
+          <p>It appears you do not have an  Earthdata Login account linked with your Launchpad account. MMT and CMR permissions (ACLs) are based on your Earthdata Login account. Please click the button below to log in to the Earthdata Login account you listed on your NAMS request and would like to associate with your Launchpad account.</p>
```
Comment (charleshuang80) [inline app/views/users/prompt_urs_association.html.erb]: It looks like there is an extra space before the first 'Earthdata Login' which is causing the test to fail

Comment (charleshuang80) [reply app/views/users/prompt_urs_association.html.erb]: Also you're using an 'an' here and the test is looking for 'a' before 'Earthdata Login'.

File: spec/features/users/user_login_spec.rb (hunk 2 above)
Comment (charleshuang80) [reply spec/features/users/user_login_spec.rb]: This test failure was fixed in one of the subsequent commits.

**Pull request merged**